Automattic/wp-calypso
# Domains: Clean old domain management code

### Changes proposed in this Pull Request

We deployed the new domain management redesigned pages for all users with #59241, but as the new components were mostly based on older components, some obsolete and unused code remained behind. This PR aims to clean that unused code.

### Testing instructions

- No behavior changes should happen because of this PR, but please do a smoke test on the domain management pages (both the site domains and all domains) to ensure everything still looks and works as expected
- Also please test the bulk contact information update page at `/domains/manage?site=all&action=edit-contact-email` and ensure it's working normally
